packages/core/protocol-config.json

```diff
@@ -6,6 +6,9 @@
       "token_contract_address": "${HARDHAT_TOKEN_CONTACT_ADDRESS}",
       "channels_contract_address": "${HARDHAT_CHANNELS_CONTACT_ADDRESS}",
       "minted_token_receiver_address": "0x2402da10A6172ED018AEEa22CA60EDe1F766655C",
+      "stake_v2_contract_address": "0x43d13D7B83607F14335cF2cB75E87dA369D056c7",
```

Comment:
I guess this is no longer needed here?

Reply:
nvm, this is needed for deployment, does it make sense to rename it to `stake_contract_address` to make it applicable for new stake versions?

Reply:
It's indeed for the deployment, but as suggested by @tolbrino, these address should be left out, because it's not relevant for the "NetworkRegistry" contract, but rather for the proxy contract. In theory, when staking v2 completes and we would introduce v3, a new proxy should be deployed and the new proxy will need v3 address.

Reply:
Why is this still part of our environments configuration?